boards/v2/raspberry_pi/rpi_pico/rpi_pico_rp2040_w.yaml

```diff
@@ -1,4 +1,4 @@
-identifier: rpi_pico_w
+identifier: rpi_pico//w
```

Comment:
Is the double slash intentional?

Reply:
Yes, the layout is `<board>/<soc>/<variant>`, because there is only one soc this will automatically become `rpi_pico/rp2040/w` by the build system